### DISCUSSION TITLE
imports codes fails

### DISCUSSION THREAD

**Discussion by henricwilkens** [ente-io/ente#4418] [Q&A]
Just installed a new mac from an old. I am trying to import the "Plain text" codes. This worked on other devices. Here it pops-up a window " Sorry Could not parse the selected file." and gives me the option to contact support. Well here I am filing this issue.

**Comment by marcvig** [comment]
Same issue on my mac.  even tried one single line plaint text import and it fails with same error.  Sorry to see you had no reply to your issue.  did you ever get it sorted out?

**Comment by henricwilkens** [comment]
I found a file to delete in the ~/Library folder after de-installing the app. Then I could reinstall and import correctly. I think it must be the file storing the encrypted secrets.On 17 Feb 2025, at 20:31, marcvig ***@***.***> wrote:﻿
Same issue on my mac.  even tried one single line plaint text import and it fails with same error.  Sorry to see you had no reply to your issue.  did you ever get it sorted out?

**Comment by henricwilkens** [comment]
from my bash history its: 

rm -rf  Library/Containers/io.ente.auth.mac

**Reply by marcvig** [reply]
I ended up brute forcing it in a way.  I used a script below to write out QR codes and then scanned all 60 of mine in.  Time consuming but it worked in the end.  Faster than manual entry of each at least.


#!/bin/bash
mkdir -p qr_codes
counter=1
while IFS= read -r line || [ -n "$line" ]; do
    # Trim leading/trailing whitespace
    trimmed_line=$(echo "$line" | xargs)
    # Only process non-empty lines
    if [ -n "$trimmed_line" ]; then
        qrencode -o "qr_codes/otp_${counter}.png" "$trimmed_line"
        echo "Generated qr_codes/otp_${counter}.png"
        counter=$((counter+1))
    fi
done < otpauth_urls.txt

I also installed "qrencode" with brew on my mac so this would work.
